recipes/diagrams/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "diagrams" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dcc03eccdfbbfb25e3804f0e13c5948292ba81b4ccf6f3a863b9467b7b5fe1ea
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not py36]
```

Comment:
```suggestion
  skip: True  # [py2k]
```

Reply:
@marcelotrevisani would this be necessary given that conda-forge no longer supports python 2.7 anyway?

Reply:
yeap, in that sense it is not necessary
I just added another suggestion which is specified in the metadata of this package, but I don't know if it is really necessary
or do you want to create a package for just python 3.6 ?

Reply:
@marcelotrevisani thank you, the selector you added helped get py37 to work!